src/Components/WindowArea/Window/Window.module.css

```diff
@@ -13,7 +13,7 @@
     0 100px 80px rgba(0, 0, 0, 0.07);
   transition: 0.2s background, 0.2s color;
   color: black;
-  animation: 0.3s Pop ease;
+  animation: pop 0.3s ease;
```

Comment:
Since CSS is case-sensitive, this should have been `Pop 0.3s ease`

Reply:
that must be an accident lol aight fixing that line